tensorflow/compiler/tf2tensorrt/convert/utils.h

```diff
@@ -81,6 +81,7 @@ string DebugString(const nvinfer1::ITensor& tensor);
 string DebugString(const std::vector<nvinfer1::Dims>& dimvec);
 string DebugString(const std::vector<TensorShape>& shapes);
 string DebugString(const std::vector<PartialTensorShape>& shapes);
+string DebugString(const nvinfer1::IBuilderConfig& config);
```

Comment:
IBuilderConfig is only available in TRT 6+ and not available in TRT5.

Reply:
Oh right. I'll protect this.